deploy/helm-chart/brudi-operator/templates/restore-crd.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: restores.mittwald.systems
+  labels:
+  {{- include "chart.labels" . | nindent 4 }}
```

Comment:
templating does not work if moved to the `crds` directory.
the default helm labels are not required anyway and can be removed